cli/tauri.js/src/template.ts

```diff
@@ -52,16 +74,19 @@ Run \`tauri init --force template\` to overwrite.`)
   }
 }
 
-/**
- *
- * @param {string} injectPath
- * @param {string} type ['conf'|'template'|'all']
- * @param {string|boolean} [force=false] - One of[false|'conf'|'template'|'all']
- * @param {boolean} [logging=false]
- * @param {string} [tauriPath=null]
- * @returns {boolean}
- */
-const inject = (injectPath, type, { force = false, logging = false, tauriPath = null }) => {
+const inject = (
+  injectPath: string,
+  type: 'conf' | 'template' | 'all',
+  {
+    force = false,
+    logging = false,
+    tauriPath
+  }: {
```

Comment:
An example of a duplicate a type definition, from my above comment on inline type definitions.

Reply:
Done! Thanks for the suggestion! 437ce947f165a0db38b6383c9d25afc0d26cc274